CMakeLists.txt

```diff
@@ -314,34 +314,28 @@ endif (NOT MACOSX)
 find_package (OpenSSL REQUIRED)
 set (INCLUDE_DIRECTORIES ${INCLUDE_DIRECTORIES} ${OPENSSL_INCLUDE_DIR})
 
-find_package (GTest REQUIRED)
-set (INCLUDE_DIRECTORIES ${INCLUDE_DIRECTORIES} ${GTEST_INCLUDE_DIRS})
+if (BUILD_TESTING OR BUILD_UNITTESTS OR BUILD_UNITTESTS_DEBUG)
```

Comment:
Which targets do you mean? Everything compiled here, but it may be that it's just because headers are in `/usr/include`. I will uninstall GTest to make sure that `find_package(GTest)` is not needed when not compiling tests.